tensorflow_datasets/testing/test_data/fake_examples/en_tam_parallel_text/en-ta-parallel-v2/corpus.bcn.dev.ta

```diff
@@ -0,0 +1,2 @@
+மணிச்சட்டம்
+குறுக்கு முகவரியிடல்
```

Comment:
As here, you have to finish at the end of the line.

Reply:
Thank you @us . will finish the suggested changes and let you know.